03-nodejs/index.js

```diff
@@ -29,4 +29,45 @@ const app = express();
 
 // TODO
 
+//1 - create the GET Users endpont
+app.get('/users', function(req, res) {
+  //Set the response header to download csv file
+  usersCall((err, results) => {
+    if (err) {
+      res.send('error');
+    }
+    res.setHeader('Content-Type', 'text/csv');
+    res.setHeader(
+      'Content-Disposition',
+      'attachment; filename="' + 'users-' + Date.now() + '.csv"'
+    );
+    res.send(formatCSVOutput(results));
+  });
+});
+
+const usersCall = function(cb) {
+  User.find()
+    .limit(2)
+    .exec(function(err, users) {
+      cb(err, users);
+    });
+};
+
+const formatCSVOutput = function(users) {
+  const output = [];
+
```

Comment:
TechDebt: I possible could have got the headers from the mongodb query, using the aggregation framework.